azkaban-db/src/main/sql/create.project_flow_files.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE project_flow_files (
+  project_id        INT          NOT NULL,
+  project_version   INT          NOT NULL,
+  flow_name         VARCHAR(128) NOT NULL,
```

Comment:
is 128 our convention?

Reply:
We should double check on this.

Reply:
Yes, in our current DB table, flow_id/flow_name is 128.

Reply:
Discussed offline. We can remain it to be 128 as same as flow_id in other tables.